CONTRIBUTING.md

```diff
@@ -292,6 +292,13 @@ New versions of packages are published on [CHaP].  To release packages to
   state published in `CHaP`.  One must resolve all compilation issues before
   merging the `CHaP` branch.  On a successful run, the script will add a comment
   on the `CHaP` PR.
+* After the versions were published to `CHaP`, push the tags created by
+  `./script/release-to-chap.sh` to `origin`.  Usually this command will push
+  all the tags:
+  ```
+  git push origin $(git tag --points-at=HEAD)
+  ```
+* Update the [release board].
```

Comment:
Want to give a rough idea or instructions on how to do this? Perhaps using either the script or the cli incantation?

Reply:
it's a manual process now; I hope in mid-term (within a few releases) we'll have a script which can do that we should add it.